xbmc/application/ApplicationVolumeHandling.cpp

```diff
@@ -139,9 +139,9 @@ void CApplicationVolumeHandling::CacheReplayGainSettings(const CSettings& settin
 {
   // initialize m_replayGainSettings
   m_replayGainSettings.iType = settings.GetInt(CSettings::SETTING_MUSICPLAYER_REPLAYGAINTYPE);
-  m_replayGainSettings.iPreAmp = settings.GetInt(CSettings::SETTING_MUSICPLAYER_REPLAYGAINPREAMP);
+  m_replayGainSettings.iPreAmp = static_cast<float>(settings.GetNumber(CSettings::SETTING_MUSICPLAYER_REPLAYGAINPREAMP));
```

Comment:
GetNumber is double and you cast to float, but the iPreAmp is integer
not so good here